lynx/patterns/link-in-bio-audio embed.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Title: Link in Bio with audio embed
+ * Slug: lynx/link-in-bio-audio embed
```

Comment:
Is this a valid slug? I think it's missing a hyphen.
```suggestion
 * Slug: lynx/link-in-bio-audio-embed
```

Reply:
You're right about the missing dash, but oddly enough the pattern still shows up for me in the inserter ¯\_(ツ)_/¯